js/siteorigin-panels/view/row.js

```diff
@@ -21,10 +21,9 @@ module.exports = Backbone.View.extend( {
 		var rowCells = this.model.get('cells');
 		rowCells.on( 'add', this.handleCellAdd, this );
 		rowCells.on( 'remove', this.handleCellRemove, this );
```

Comment:
Can also use `this.listenTo` here.

Reply:
Same case here. Isn't `listenTo` only for listening to events on other objects?

Reply:
My bad, rad, will fix that in develop.